### PR TITLE
Record timings in Ophan

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Currently, only a limited set of performance metrics are recording in Ophan. A request is constructed and sent in [Ophan's Tracker JS](https://github.com/guardian/ophan/blob/f118c862a9a32bbded33acb0a928049d269724a1/tracker-js/assets/coffee/ophan/perf.coffee#L29) when the page load event is fired.

This change fires an additional request, after the Ophan request described above. This contains additional performance metrics specific to the way the frontend JS is loaded and executed.

## What is the value of this and can you measure success?

This will allow us to capture performance metrics at a more granular level than the vanilla Ophan request provides. It will capture metrics for all traffic, not just the 0.1% captured by Google Analytics. This will inform better decisions regarding architectural changes (e.g. migration to Webpack).

## Does this affect other platforms - Amp, Apps, etc?

No

## Tested in CODE

No

## TODO

- [x] merge guardian/ophan#1984 to events.thrift in Ophan

### CODE REVIEW

**Comment by SiAdcock** [comment]
I understand I'll have to update the names and positions of the performance marks based on #15700

**Review by sndrs — DISMISSED** [review]
great!

File: static/src/javascripts-legacy/projects/common/utils/capture-perf-timings.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            firstByte: timing.responseStart - timing.connectEnd,
+            lastByte: timing.responseEnd - timing.responseStart,
+            domContentLoadedEvent: timing.domContentLoadedEventStart - timing.responseEnd,
+            loadEvent: timing.loadEventStart - timing.domContentLoadedEventStart,
+            navType: window.performance.navigation.type,
+            redirectCount: window.performance.navigation.redirectCount,
+            assetsPerformance: marks.map(function (mark) {
+                return {
+                    name: mark,
+                    timing: parseInt(userTiming.getTiming(mark) || 0),
```
Comment (sndrs) [inline static/src/javascripts-legacy/projects/common/utils/capture-perf-timings.js]: need a radix

**Comment by sndrs** [comment]
not sure what i've done with my review now! but 👍 with the radix sorted – great stuff, really looking forward to using it

**Pull request merged**